src/azul/service/elasticsearch_service.py

```diff
@@ -96,6 +96,9 @@ def _translate_filters(self, filters: Filters, field_mapping: JSON) -> MutableFi
             assert isinstance(value, dict)
             assert isinstance(one(value.values()), list)
             field_type = self.field_type(tuple(key.split('.')))
+            if key == 'organismAge':
+                # Special handling required; https://github.com/DataBiosphere/azul/issues/1747#issuecomment-642247817
+                value = {k: [f'{d["value"]} {d["unit"]}' for d in v] for k, v in value.items()}
```

Comment:
```suggestion
                value = {k: [f'{d["value"]} {d["unit"]}'.strip() for d in v] for k, v in value.items()}
```
Unless 'value' and 'unit' will never be empty in this context.
Also, since this is special handling of a filter param there should probably be a test case in `test_response.py` that uses this filter param.